sephora/game/custom_turfs.dm

```diff
@@ -21,13 +61,24 @@
 	desc = "A padded piece of hull plating which can make a space based installation feel more homely."
```

Comment:
This is an airlock not a carpet

Reply:
Are you sure about that? Lines 59 -74 are the carpet. Lines 49-57 are the airlock.